Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -582,94 +573,9 @@ extension Channel {
             http2ConnectionInitializer: http2ConnectionInitializer
         )
     }
-
-    /// Configures a `ChannelPipeline` to speak HTTP/2 and wraps both the connection channel and any
-    /// created inbound stream channels in `NIOAsyncChannel`s.
-    ///
-    /// In general this is not entirely useful by itself, as HTTP/2 is a negotiated protocol. This helper does not handle negotiation.
-    /// Instead, this simply adds the handler required to speak HTTP/2 after negotiation has completed, or when agreed by prior knowledge.
-    /// Use this function to setup a HTTP/2 pipeline if you wish to use `NIOAsyncChannel`s to provide async sequence abstractions
-    /// over inbound and outbound streams whilst handling back-pressure.
-    /// 
-    /// Using this rather than implementing a similar function yourself allows that pipeline to evolve without breaking your code.
-    ///
-    /// - Parameters:
-    ///   - mode: The mode this pipeline will operate in, server or client.
-    ///   - configuration: The settings and initialization configuration for both connections and streams, and their wrapping
-    ///     ``NIOAsyncChannel``s.
-    /// - Returns: An `EventLoopFuture` containing the `AsyncStreamMultiplexer` inserted into this pipeline which wraps
-    ///     inbound streams as `NIOAsyncChannels` after initialization. The multiplexer can be used to initiate new streams
-    ///     and iterate over inbound HTTP/2 stream channels.
-    @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
-    @_spi(AsyncChannel)
-    public func configureAsyncHTTP2Pipeline<ConnectionInbound, ConnectionOutbound, StreamInbound, StreamOutbound>(
```

Comment:
Why are we removing this variant with this change? It's not related to protocol negotiation.